src/openklant/components/klantinteracties/models/digitaal_adres.py

```diff
@@ -36,6 +37,7 @@ class DigitaalAdres(APIMixin, models.Model):
             "met dit adres contact kan worden opgenomen."
         ),
         max_length=255,
+        choices=SoortDigitaalAdres.choices,
```

Comment:
The issue also mentions:

> Add an 'overig' (other) option with the option to add a custom type of digital address.

So we might have to add an extra field in which this custom type can be specified?

Reply:
Might be because of the wording but I think it refers to having an `overig` option to allow the `adres` field to contain other values than that of the `email`/`telefoonnummer` types.